### PR TITLE
docs: add documentation for methods used to mount an UFS path to an Alluxio path

### PR DESCRIPTION
**Pull request by abnerferreira0212**
# Pull Request Merge Checklist
           
Task backlog identifier related to this PR: 1542

---

## As a DEVELOPER, I confirm that this PR **doesn't contain**:
- [x] Code lines with syntax errors
- [x] Methods and variables names that aren't clear or doesn't mean anything
- [x] Code that is out of the standard (Example: **PEP-8** for Python code)
- [x] Files with **imports** errors or without any utility
- [x] **TODOs** left behind
- [x] Tests that are being skipped
- [x] Methods placed on incorrect files or folders (Example: methods that are local, but should be placed on /utils)
- [x] Functions, methods or API responses that are publicly returning sensitive information

## As a DEVELOPER, I confirm that I:
- [x] Followed the Simbiose's standard for branch and commit naming
- [x] Concluded the task and achieved the goals according to the task planning I defined before starting to code
- [ ] Added all necessary database changes to the [SQL Migrations](https://github.com/s1mbi0se/sql-migrations)
- [ ] Created tests for every new flow and method on this Pull Request
- [ ] Added the necessary DEBUG logs
- [x] Checked my code using the Simbiose's Corner Cases Guide
- [ ] Merged the **develop** branch to this PR/branch before sending the PR for review
- [x] Solved all possible merge conflicts before sending the PR for review
- [ ] Corrected all the problems and code smells identified by the quality tools
- [ ] Manually executed the necessary tests locally to make sure my changes didn't break anything
- [ ] All unitary and integration tests ran by the CI system are passing
- [ ] Check the performance of my changes or additions and they didn't make the system slower (I profiled the code)
- [x] Updated the necessary documentation (Example: OpenAPI, Docstrings, Metadata Database Modeling)
- [x] Made the self-review of the code before sending the PR for review
- [ ] Sent this PR for a peer-review
- [x] Made all the corrections appointed during the peer-review
- [x] Confirmed that this PR merge will not cause other systems or dependencies to stop working (Example: ShannonDB Python Client and ShannonDB)
- [ ] Checked all GitHub Actions related to this PR that can block my merge

---  

## As the TEAM MANAGER, I confirm that I:
- [ ] Checked that the code was developed following the best practices for the language, with the best possible readability and using the less possible quantity of code lines and complexity. 
- [ ] Checked if the Pull Request and commit messages follow the Simbiose’s defined standard 
- [ ] Checked if there is no possible merge conflicts before reviewing the PR
- [ ] Checked if the purpose of the task was reached with this PR
- [ ] Checked that the code was developed following the best practices for the language, with the best possible readability and using the less possible quantity of code lines and complexity. 
- [ ] Checked if all related PRs were created when other components were affected by this task (e.g. database changes on [SQL Migrations](https://github.com/s1mbi0se/sql-migrations))
- [ ] Checked that were created unit, integration and chaos tests for every new flow and method on this Pull Request
- [ ] Checked that no sensible information (e.g. users security data, internal security data, etc) was exposed by this PR
- [ ] Checked that code coverage for this PR is higher than 80%.
- [ ] Checked that there are no problems and code smells identified by the quality and security tools
- [ ] Checked that all files were created on the proper folders according to the Simbiose’s defined standard
- [ ] Checked that all classes, methods and variables on this PR follow the Simbiose’s pattern for this programming language
- [ ] Checked that all unitary and integration tests ran by the CI system are passing
- [ ] Checked that there are no performance issues created by these changes or additions
- [ ] Checked that all documentation is up to date according to this change (Example: OpenAPI, Docstrings, Metadata Database Modeling)
- [ ] Checked that this PR merge will not cause other systems or dependencies to stop working (Example: ShannonDB Python Client and ShannonDB)
- [ ] Checked all GitHub Actions related to this PR that can block the merge

---  

## As the QUALITY RESPONSIBLE, I confirm that I:
- [ ] Checked if the Pull Request and commit messages follow the Simbiose’s defined standard 
- [ ] Checked if there is no possible merge conflicts before reviewing the PR
- [ ] Checked that were created unit, integration and chaos tests for every new flow and method on this Pull Request
- [ ] Checked that no sensible information (e.g. users security data, internal security data, etc) was exposed by this PR
- [ ] Checked that code coverage for this PR is higher than 80%.
- [ ] Checked that there are no problems and code smells identified by the quality and security tools
- [ ] Checked that all files were created on the proper folders according to the Simbiose’s defined standard
- [ ] Checked that all classes, methods and variables on this PR follow the Simbiose’s code standard for this programming language
- [ ] Checked that all unitary and integration tests ran by the CI system are passing
- [ ] Checked that there are no performance issues created by these changes or additions
- [ ] Checked that all documentation is up to date according to this change (Example: OpenAPI, Docstrings, Metadata Database Modeling)
- [ ] Checked that this PR merge will not cause other systems or dependencies to stop working (Example: ShannonDB Python Client and ShannonDB)
- [ ] Checked all GitHub Actions related to this PR that can block the merge

## Important links:
1. [Simbiose Quality Guidelines](https://github.com/s1mbi0se/company/wiki/Quality-Topics#table-of-contents)
2. [Metadata Database Modeling](https://docs.google.com/spreadsheets/d/1Y8_k5xV86kTjgfjbi-PGsJcubUKhWlT2x8ecpnMlyR0/edit?usp=sharing)
3. [Simbiose's Corner Cases Guide](https://www.simbioseventures.com/documentos-e-guias/procedimentos-e-checklists/corner-cases-guide)

### CODE REVIEW

File: core/common/src/main/java/alluxio/metrics/MetricsSystem.java [hunk 1]
```diff
@@ -173,7 +173,32 @@ public static void startSinks(String metricsConfFile) {
   }
 
   /**
-   * Constructs the source name of metrics in this {@link MetricsSystem}.
+   * Constructs and returns the source name of metrics in this metrics system.
+   * <p>
+   * Sets {@code sourceKey} according to the value of {@link CommonUtils#PROCESS_TYPE}.
```
Comment (caiopavanelli) [inline core/common/src/main/java/alluxio/metrics/MetricsSystem.java:178]: I believe that it's unnecessary to reproduce the switch statement in Javadoc. Keep ip simple for the essence of the method, not the syntax.

File: core/common/src/main/java/alluxio/util/CommonUtils.java [hunk 2]
```diff
@@ -766,11 +766,15 @@ public static String convertMsToClockTime(long millis) {
...
-   * Memoize implementation for java.util.function.supplier.
-   *
-   * @param original the original supplier
-   * @param <T> the object type
-   * @return the supplier with memorization
+   * Updates and returns supplier, implementing memoization
+   * <p>
+   * Memoize implementation for {@link Supplier<T>}.
+   * <p>
+   * Memoize implementation for the Java supplier.
```
Comment (caiopavanelli) [inline core/common/src/main/java/alluxio/util/CommonUtils.java:773]: redundant

File: core/common/src/main/java/alluxio/util/network/NettyUtils.java [hunk 3]
```diff
@@ -188,13 +188,27 @@ public static ChannelType getWorkerChannel(AlluxioConfiguration conf) {
...
 
   /**
-   * Get the proper channel class.
+   * Gets the proper channel class based on the provided property key and Alluxio configuration.
+   * <p>
+   * Checks if domain socket is enabled and channel type is {@link ChannelType#EPOLL}. Throws an
+   * exception if true. Returns {@code EpollDomainSocketChannel.class} if domain socket is enabled
+   * but channel type is not EPOLL.
+   * <p>
+   * If domain socket is disabled, checks whether {@code channelType} is {@link ChannelType#NIO}
```
Comment (caiopavanelli) [inline core/common/src/main/java/alluxio/util/network/NettyUtils.java:197]: Summarize this paragraph.

File: core/common/src/main/java/alluxio/util/CommonUtils.java (hunk 2 above)
Comment (abnerferreira0212) [reply core/common/src/main/java/alluxio/util/CommonUtils.java]: Fixed.

File: core/common/src/main/java/alluxio/util/network/NettyUtils.java (hunk 3 above)
Comment (abnerferreira0212) [reply core/common/src/main/java/alluxio/util/network/NettyUtils.java]: Fixed,

File: core/common/src/main/java/alluxio/metrics/MetricsSystem.java (hunk 1 above)
Comment (abnerferreira0212) [reply core/common/src/main/java/alluxio/metrics/MetricsSystem.java:178]: Fixed.

File: core/common/src/main/java/alluxio/master/PollingMasterInquireClient.java [hunk 4]
```diff
@@ -98,6 +98,20 @@ public PollingMasterInquireClient(List<InetSocketAddress> masterAddresses,
...
+  /**
+   * Gets the primary master RPC address.
+   * <p>
+   * Iterates through each {@link InetSocketAddress} in
+   * {@link #mConnectDetails} and tries to connect to
+   * one of them following the retry policy from {@link #mRetryPolicySupplier}.
+   * Returns the address of the successfully made connection. Throws an exception
+   * if no connection is established.
+   *
+   * @return  the INET socket address for the
```
Comment (joaopedro-simbiose) [inline core/common/src/main/java/alluxio/master/PollingMasterInquireClient.java:110]: Finish the phrase please
